Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -1413,12 +1413,21 @@ namespace AtomSampleViewer
 
     bool ScriptManager::PrepareForScreenCapture(const AZStd::string& imageName)
     {
-        AZStd::string fullFilePath;
+        AZ::Render::FrameCapturePathOutcome pathOutcome;
+
         AZ::Render::FrameCaptureTestRequestBus::BroadcastResult(
-            fullFilePath,
+            pathOutcome,
             &AZ::Render::FrameCaptureTestRequestBus::Events::BuildScreenshotFilePath,
             imageName, true);
 
+        if (!pathOutcome.IsSuccess())
+        {
+            ReportScriptError(pathOutcome.GetError().m_errorMessage);
+            return false;
+        }
+
+        AZStd::string fullFilePath = pathOutcome.IsSuccess() ? pathOutcome.GetValue() : "";
```

Comment:
nit: no real need for the ternary operator, pathOutcome has already been checked for success so it can be assumed.

Can wait for any further work.